app/src/db/migrations/20200616175436_014-agriseafoodopscreening-change-sleeping-fields.js

```diff
@@ -5,20 +5,23 @@ exports.up = function (knex) {
   return Promise.resolve()
     .then(() => knex.schema.alterTable(`${AGRI_PREFIX}_submission_attestation`, table => {
       table.boolean('sharedSleepingCommunication').notNullable().defaultTo(false);
+      table.boolean('sharedSleepingProvidedAccomodations').notNullable().defaultTo(false);
 
       table.dropColumn('sharedSleepingPerRoom');
       table.dropColumn('sleepingAreaType');
       table.dropColumn('commonAreaDistancing');
     }))
     .then(() => knex.schema.alterTable(`${AGRI_PREFIX}_submission_attestation`, table => {
-      table.boolean('sharedSleepingCommunication').notNullable().alter();
```

Comment:
shouldn't this still be in the script?  we need to run it for both new columns.

Reply:
Just fixed this